modules/backend/routes.php

```diff
@@ -25,7 +25,7 @@
             'middleware' => ['web'],
             'prefix' => Config::get('cms.backendUri', 'backend')
         ], function () {
-            Route::any('{slug}', 'Backend\Classes\BackendController@run')->where('slug', '(.*)?');
+            Route::any('{slug?}', 'Backend\Classes\BackendController@run')->where('slug', '(.*)?');
```

Comment:
Is this a change that needs to be documented? It affected RainLab.Translate so the possibility exists that other plugins could be affected.

Reply:
@LukeTowers done - https://github.com/octoberrain/meta/commit/81507a443a9027fcb9a03bb82b2b2b3f7c1652fb.